prestoadmin/collect.py

```diff
@@ -107,9 +107,9 @@ def file_get(remote_path, local_path):
 
     _LOGGER.debug('local path used ' + path_with_host_name)
 
-    if exists(remote_path, True):
+    try:
         get(remote_path, path_with_host_name, True)
-    else:
+    except SystemExit:
         warn('remote path ' + remote_path + ' not found on ' + env.host)
 
```

Comment:
Can we change the name of this method to get_files to signify that there could be one or more files being fetched?